libs/core/futures/include/hpx/futures/traits/detail/future_await_traits.hpp

```diff
@@ -194,7 +201,7 @@ namespace hpx { namespace lcos { namespace detail {
 }}}    // namespace hpx::lcos::detail
 
 ///////////////////////////////////////////////////////////////////////////////
-namespace std {
+namespace coro {
```

Comment:
This has issues if the `coro` is `std::experimental`, any suggestions?

Reply:
What issues?

Reply:
https://pastebin.com/2hmJjAEU @hkaiser

Reply:
We currently specialize `coroutine_traits` in `namespace std` (here: https://github.com/STEllAR-GROUP/hpx/blob/master/libs/core/futures/include/hpx/futures/traits/detail/future_await_traits.hpp#L197). If you add support for coroutines that are in `std::experimental`, this specialization will need to go into that namespace as well.

Reply:
What I meant is that you need to explicitly say what 'original' namespace the coroutine_traits base template is defined:
```
namespace HPX_CORO_NAMESPACE {
...
}
```
where `HPX_CORO_NAMESPACE` is either `std` or `std::experimental`.